Test/prover/cvc5.bpl

```diff
@@ -0,0 +1,8 @@
+// RUN: %boogie /trace /proverOpt:SOLVER=cvc5 "%s" > "%t"
```

Comment:
I bet after the last review you know what I'm about to ask here. :)

In this case I'm fine with punting deeper coverage for now, but I'm at least curious what percentage of the suite passes with CVC5.

Reply:
In the brief attempts I've made, most of the tests time out. So I think it may be too early to try running them all in CI just yet. The changes for included for testing batch mode will make it easy to run a separate job just testing CVC5 once that's expected to complete in a reasonable time, though.